Mlem/App/Utility/Extensions/Content Models/Post1Providing+Extensions.swift

```diff
@@ -17,4 +18,42 @@ extension Post1Providing {
             saveAction
         ])
     }
+    
+    func taggedTitle(communityContext: (any Community3Providing)?) -> Text {
+        let hasTags: Bool = removed
+            || pinnedInstance
+            || (communityContext != nil && pinnedCommunity)
+            || locked
+        
+        return postTag(active: removed, icon: Icons.removeFill, color: Palette.main.negative) +
+            postTag(active: pinnedInstance, icon: Icons.pinFill, color: Palette.main.administration) +
+            postTag(active: communityContext != nil && pinnedCommunity, icon: Icons.pinFill, color: Palette.main.moderation) +
+            postTag(active: locked, icon: Icons.lockFill, color: Palette.main.orange) +
```

Comment:
Yeah, that makes sense. I originally had it as a passthrough literal because "lock" is such a narrow scope to tie a color to compared to more ubiquitous concepts like upvotes, downvotes, moderation, etc; I've renamed this to `secondaryAccent`, since we use it as basically a miscellaneous "other" color (lock, collapse comment swipe background, OP flair--I can't make a coherent group out of those)